batch/filter.go

```diff
@@ -0,0 +1,51 @@
+package batch
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/minio/pkg/wildcard"
+)
+
+type Filter struct {
+	Name   string
+	Values []string
+}
+
+func filterJSON(data []map[string]interface{}, filters []*Filter) ([]map[string]interface{}, diag.Diagnostics) {
```

Comment:
Comment what is `[]map[string]interface{}` representing? What's the result?

Reply:
Done!